### PR TITLE
devcontainer: use VirtualGL on mac hosts

### PR DESCRIPTION
**Pull request by fredyshox**
Use VirtualGL to force direct rendering of OpenGL (as opposed to indirect rendering which happens by default while forwarding X with XQuartz). This should cameraview in selfdrive/ui/ui and tools like plotjuggler/cabana.

### CODE REVIEW

File: tools/README.md [hunk 1]
```diff
@@ -50,6 +50,19 @@ Dev containers are supported in [multiple editors and IDEs](https://containers.d
...
+
+Some GUI tools require OpenGL to work properly. To use them with XQuartz on mac, VirtualGL needs to be installed.
+
+```
+brew install --cask virtualgl
+```
+
+Then when invoking GUI tool (e.g. ui, cabana or plotjuggler), prefix the command with `vglrun`. For instance:
+```
+vglrun python3 tools/plotjuggler/juggle.py --demo
```
Comment (adeebshihadeh) [inline tools/README.md:63]: can we do this by default instead of adding all this stuff to the README?

Comment (fredyshox) [reply tools/README.md:63]: `vglrun` is a shell script, so I guess i could source it in bashrc instead. But it's messing with LD_* paths, and im not sure how it affects non-UI stuff

Comment (adeebshihadeh) [reply tools/README.md:63]: it's very much worth looking into that. just having it work is 100x better than running a weird command

Comment (fredyshox) [reply tools/README.md:63]: i would leave it as is for today, and try embedding it in bashrc once we have some more tests for devcontainer

Comment (adeebshihadeh) [reply tools/README.md:63]: which libraries does it touch? I assume only GL stuff?

Comment (fredyshox) [reply tools/README.md:63]: This is the implementation: https://github.com/VirtualGL/virtualgl/blob/main/server/vglrun.in

Apart from adding paths to `LD_LIBRARY_PATH`. Its filling up `LD_PRELOAD` with faker libraries which has special implementations of some of the system funtions like`dlopen` and `getenv`.

File: tools/README.md [hunk 2]
```diff
@@ -50,6 +50,14 @@ Dev containers are supported in [multiple editors and IDEs](https://containers.d
 
 GUI apps like `ui` or `cabana` can also run inside the container by leveraging X11 forwarding. To make use of it on macOS, additional configuration steps must be taken. Follow [these](https://gist.github.com/sorny/969fe55d85c9b0035b0109a31cbcb088) steps to setup X11 forwarding on macOS.
 
+#### VirtualGL on macOS
+
+Some GUI tools require OpenGL to work properly. To use them with XQuartz on mac, VirtualGL needs to be installed.
```
Comment (adeebshihadeh) [inline tools/README.md:55]: can we just print this as a MOTD style thing when the devcontainer starts up if we detect it's not setup?

Comment (fredyshox) [reply tools/README.md:55]: sure

**Pull request merged**